### PR TITLE
GUI - SFML

### PR DESCRIPTION
**Pull request by osvegn**
It adds the SFML part. It contains the main loop, the event handling, and a little fix in a CMakeLists.txt of the event part.
Link to the issue: #68.

### CODE REVIEW

**Review by LucasTesnier — APPROVED** [review]
NTR

**Review by GuyomT — APPROVED** [review]
LGTM

File: src/zappy_gui/Core/SFML/SFML.cpp [hunk 1]
```diff
@@ -8,3 +8,69 @@
...
+    _window.get()->clear(sf::Color(127, 127, 127, 255));
+    _window.get()->display();
+}
+
+void SFML::_getEvent()
+{
+    sf::Event event;
+
+    while (_window.get()->pollEvent(event)) {
+        if (event.type == sf::Event::Closed) {
```
Comment (pariciadra) [inline src/zappy_gui/Core/SFML/SFML.cpp:33]: Welcome to if land

**Review by pariciadra — APPROVED** [review]
LGTM

**Pull request merged**